export_repos.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import os
+import shutil
+import gitstats
```

Comment:
Please, recheck if it gets imported, I wonder if we need to to
`import .gitstats`

Reply:
Hi I try in win and linux OS too, but doeasn't works this import without .py ext.